app/controllers/magic_link/sign_up_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MagicLink::SignUpController < ApplicationController
+  def new; end
+
+  def create
+    candidate = Candidate.new(candidates_params)
+    MagicLinkFlow.sign_up(candidate)
+
+    redirect_to check_your_emails_path
+  end
+
+  def show; end
+
+private
+
+  def candidates_params
```

Comment:
This is only a single candidate so could be `candidate_params`

Reply:
Makes sense, we've updated this.